app/controllers/case_logs_controller.rb

```diff
@@ -128,4 +128,19 @@ def api_case_log_params
 
     params.require(:case_log).permit(CaseLog.editable_fields)
   end
+
+  def get_next_page_path(form, previous_page, responses_for_page = {})
+    questions_for_page = form.questions_for_page(previous_page)
+    questions_for_page.each do |question, content|
+      next unless content.key?("conditional_route_to")
```

Comment:
Hypothetically if multiple questions on the same page have a conditional_route_to, am I right in thinking this will just return the result for the 1st one?

That's probably fine but it might be more explicit to raise an error if that's been defined?